### ISSUE TITLE
Losses decrease quite slowly with custom dataset

### ISSUE THREAD

**Issue by bbsvip** [Karel911/TRACER#31]
I created a custom dataset and tried training with arch 5 with image size 512 but the loss decreased quite slowly. Is this normal, or is my data problem?

my loss
Epoch:[001/200]
Train Loss:99.721 | MAE:0.189
Valid Loss:82.747 | MAE:0.125
=============================Epoch:[002/200]
Train Loss:91.156 | MAE:0.156
Valid Loss:91.560 | MAE:0.166
=============================Epoch:[003/200]
Train Loss:91.532 | MAE:0.155
Valid Loss:78.634 | MAE:0.106
=============================Epoch:[004/200]
Train Loss:86.963 | MAE:0.135
Valid Loss:76.247 | MAE:0.095
=============================Epoch:[005/200]
Train Loss:85.692 | MAE:0.130
Valid Loss:74.397 | MAE:0.086
=============================Epoch:[006/200]
Train Loss:85.188 | MAE:0.127
Valid Loss:74.988 | MAE:0.086
=============================Epoch:[007/200]
Train Loss:84.972 | MAE:0.125
Valid Loss:73.414 | MAE:0.082
=============================Epoch:[008/200]
Train Loss:83.838 | MAE:0.121
Valid Loss:75.131 | MAE:0.087
=============================Epoch:[009/200]
Train Loss:83.403 | MAE:0.119

**Comment by Karel911** [comment]
Sorry for the late reply.
Did you try to decrease learning rate?
